docs/FAQ/how-to-deploy-bold-bi-in-eks-using-application-load-balancer.md

```diff
@@ -0,0 +1,220 @@
+# How to deploy Bold BI in Elastic Kubernetes Services (EKS) using Application Load Balancer (ALB)
+
+The section explains how to deploy [Bold BI](https://www.boldbi.com/) in an EKS cluster using an [Application Load Balancer](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/introduction.html).
+
+1. Create an Amazon EKS cluster and [node group](https://docs.aws.amazon.com/eks/latest/userguide/eks-compute.html) to deploy Bold BI.
+
+   https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html 
+
+2. Create an Amazon Elastic File System (EFS) volume to store the shared folders for application usage by following the link below.
```

Comment:
2. Create an Amazon Elastic File System (EFS) volume to store the shared folders for application usage by following this link.

Reply:
concern addressed